backend/src/main/resources/schema.graphqls

```diff
@@ -189,8 +189,18 @@ type PatientLink {
 
 type Query {
   deviceType: [DeviceType]
-  patients(facilityId: ID, pageNumber: Int, pageSize: Int, showDeleted: Boolean, namePrefixMatch: String): [Patient]
-  patientsCount(facilityId: ID, showDeleted: Boolean, namePrefixMatch: String): Int
+  patients(
```

Comment:
Hmm, can't tell if this is my IDE auto-applying spotless correctly or auto-applying something else incorrectly